frida_tools/application.py

```diff
@@ -5,7 +5,7 @@
 import collections
 import errno
 import numbers
-from optparse import OptionParser
+import argparse
```

Comment:
These should be kept alphabetically sorted.

Reply:
Fixed, maybe we could add isort to the CI? Maybe with the migration to Python 3 we can add Black

Reply:
That would be great!